### ISSUE TITLE
HomeAssistant warning "Payload is not supported"

### ISSUE THREAD

**Issue by stast1** [devWaves/SwitchBot-MQTT-BLE-ESP32#31]
Log
```
2021-09-29 21:36:51 WARNING (MainThread) [homeassistant.components.mqtt.cover] Payload is not supported (e.g. open, closed, opening, closing, stopped): PAUSE
```
Please add stat_stopped: PAUSE to config topic to remove this warning.
```
client.publish((home_assistant_mqtt_prefix + "/cover/" + deviceName + "/config").c_str(), ("{\"~\":\"" + (curtainTopic + deviceName) + "\", " +
                 ...
                 + "\"stat_stopped\": \"PAUSE\", " +
                 ...
```
Thanks!

**Comment by devWaves** [comment]
okay that makes sense. I don't have curtains. You're the first to mention it. Thanks for the input

I will release a v6.3 later today

**Comment by JoepWi** [comment]
Thanks. I just didn't had the time  yet to post this issue here!

**Comment by devWaves** [comment]
all good now. thanks for noticing the issue. fixed in v6.3
